### PR TITLE
Add acts_as_list gem

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by zjhuntin** [comment]
!test packaging

**Comment by xprazak2** [comment]
[test packaging]

**Comment by xprazak2** [comment]
Gem tests try to run and fail, should I make some changes so that they are not executed?

**Comment by ezr-ondrej** [comment]
@zjhuntin do we need this in deb? this will be a dependency of `foreman_ansible` so in deb the bunler will install that, right? :thinking:

File: dependencies/bionic/acts_as_list/control [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Uploaders: <>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               rake,
+               ruby-activerecord (>= 4.2)
+Standards-Version: 4.1.3
+Vcs-Git: https://salsa.debian.org/ruby-team/ruby-acts-as-list.git
+Vcs-Browser: https://salsa.debian.org/ruby-team/ruby-acts-as-list
+Homepage: http://github.com/brendon/acts_as_list
+Testsuite: autopkgtest-pkg-ruby
```
Comment (ezr-ondrej) [inline dependencies/bionic/acts_as_list/control:14]: I don't see this in any other dependencies packages. Should we comment it out?

File: dependencies/bionic/acts_as_list/ruby-tests.rake [hunk 2]
```diff
@@ -0,0 +1,6 @@
+require 'gem2deb/rake/testtask'
```
Comment (ezr-ondrej) [inline dependencies/bionic/acts_as_list/ruby-tests.rake:1]: What is this file for?

File: dependencies/bionic/acts_as_list/control [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: <>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               rake,
+               ruby-activerecord (>= 4.2)
+Standards-Version: 4.1.3
+Vcs-Git: https://salsa.debian.org/ruby-team/ruby-acts-as-list.git
+Vcs-Browser: https://salsa.debian.org/ruby-team/ruby-acts-as-list
```
Comment (ezr-ondrej) [inline dependencies/bionic/acts_as_list/control:12]: These are usually commented out elsewhere :thinking:

**Review by ezr-ondrej — COMMENTED** [review]
I've just went over other dependencies and they doesn't seem to have so many stuff as you have :thinking:

**Comment by ezr-ondrej** [comment]
#6331 is fixing the test issue (just removes the tests as we don't run them for any other dependencies)

**Comment by xprazak2** [comment]
Closing in favour of #6331, thanks!

**Pull request closed without merging**